tests/test_main.py

```diff
@@ -61,4 +61,3 @@ def test_shop_trip_output(monkeypatch):
 Monica doesn't have enough money to make purchase in any shop
 '''
     assert output == out
```

Comment:
It looks like you accidentally removed newline at the end of the file here. I'd suggest to put it back. :)

Reply:
Did you mean that it should be:
```
Monica doesn't have enough money to make purchase in any shop

'''
``` ? )

Reply:
No, i meant that it should be:
```python
assert output == out  # newline here